tests/helixperftasks.targets

```diff
@@ -1,4 +1,43 @@
-<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <UsingTask TaskName="RemoveDuplicateAssemblies" TaskFactory="CodeTaskFactory" AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.v4.0.dll" >
```

Comment:
I am very much not a fan of inline tasks.  Furthermore, there's already src\Microsoft.DotNet.Build.Tasks\RemoveDuplicatesWithLastOneWinsPolicy.cs  (in Microsoft.DotNet.Build.Tasks.dll).  If that one does not do what you want it'd be good to augment it til it did, or to add an entirely new task there to do this work.